Python/Product/Analysis/ModulePath.cs

```diff
@@ -137,6 +133,7 @@ public ModulePath(string fullname, string sourceFile, string libraryPath)
             LibraryPath = libraryPath;
         }
 
+        private static readonly string[] CompiledModuleExtensions = new[] { ".pyd", ".pyc", ".pyo", ".so", ".dyLib" };
```

Comment:
Capital `L` in `dyLib` means this never matches your current test. If it works now and didn't before, does that mean we shouldn't be testing for `dylib`?

Reply:
This is a good question. Mac FS is case-insensitive *usually* but one can have it in case-sensitive form. Linux is always case-sensitive. However, I compare with `ToLowerInvariant()`